.vscode/settings.json

```
@@ -35,7 +35,7 @@
   "cssVariables.lookupFiles": [
     "**/*.css",
     "**/*.scss",
-    "node_modules/@kong/design-tokens/dist/tokens/css/variables.css"
+    "node_modules/@kong/design-tokens/dist/tokens/css/custom-properties.css"
```

can you try changing this to a glob pattern? Unsure if you'll get too many conflicts since there are two files exported, but can you try this?
```suggestion
    "node_modules/@kong/design-tokens/**/*.css"
```

---

This doesn't work at all. Other variants I've tried, none of which worked:
- `node_modules/@kong/design-tokens/tokens/**/*.css`
- `node_modules/@kong/design-tokens/tokens/css/*.css`
- `node_modules/@kong/design-tokens/tokens/css/custom-properties.css`
- `node_modules/@kong/design-tokens/dist/tokens/css/*.css`